community/kernel/src/test/java/org/neo4j/kernel/impl/store/NeoStoresTest.java

```diff
@@ -880,7 +916,7 @@ private int validateAndCountRelationships( long node, long rel1, long rel2, int
     {
         int count = 0;
         try ( KernelStatement statement = (KernelStatement) tx.acquireStatement();
-              Cursor<NodeItem> nodeCursor = statement.getStoreStatement().acquireSingleNodeCursor( node ) )
+                Cursor<NodeItem> nodeCursor = statement.getStoreStatement().acquireSingleNodeCursor( node ) )
```

Comment:
This extra indentation looks like a mistake. Resource declarations look better when they are aligned in a try-with-resources statement.